dev-proxy-plugins/MockResponses/MockResponsePlugin.cs

```diff
@@ -215,20 +215,14 @@ private void ProcessMockResponse(ProxyRequestArgs e, MockResponse matchingRespon
             statusCode = (HttpStatusCode)matchingResponse.Response.StatusCode;
         }
 
-        if (matchingResponse.Response?.Headers is not null)
+        if (matchingResponse.Response?.ResponseHeaders is not null)
         {
-            foreach (var key in matchingResponse.Response.Headers.Keys)
+            foreach (var key in matchingResponse.Response.ResponseHeaders.SelectMany(dict => dict.Select(kv => new HttpHeader(kv.Key, kv.Value))))
```

Comment:
This is no longer `key` but a whole header, right? Let's rename the variable to make it clear what we're working with